### DISCUSSION TITLE
Is there a way to make fzf-tab previews work with this plugin?

### DISCUSSION THREAD

**Discussion by 2KAbhishek** [marlonrichert/zsh-autocomplete#466] [Q&A] [answered]
Here's my [zsh config](https://github.com/2KAbhishek/Dotfiles/blob/fd0229359c92f0a6649c99048b1fedb5ae805f77/.zshrc#L71)

If you visit the link you can see that I have [fzf-tab](https://github.com/Aloxaf/fzf-tab) previews enabled.

I am using the ',' key to trigger tab-completions and preview but if I add the autocomplete plugin it doesn't work.

I see the following message, when I try to do something like `vim ,<Tab>`

```bash
no matching `file', `directory', or `ancestor directory' completions
```

The expected behaviour is that it will open up a list of files with fuzzy search and previews.

![Screenshot_1408_1](https://user-images.githubusercontent.com/25423853/184541587-5b0c4182-f566-49e5-91c3-ac0e4db5338c.png)

**Comment by marlonrichert** [comment]
Yes, it's possible, but it's not trivial. You'd need to do [something similar to this](https://github.com/marlonrichert/zsh-autocomplete/blob/4c363ff553be227db90842d7481fd6e02de8703c/scripts/.autocomplete.key-binding#L20), but for `fzf-tab`'s version of the tab widget. Pull requests welcome! 🙂

**Reply by 2KAbhishek** [reply]
alright, I'll look into it, thanks!

**Reply by Asmageddon** [reply]
Any progress on the front? :D

**Reply by marlonrichert** [reply]
@2KAbhishek?

**Reply by 2KAbhishek** [reply]
Hey, I did start on this and was looking into readline and fzf widgets, but haven't spent enough time to dive deeper and raise a PR.

**Reply by marlonrichert** [reply]
If you have any code yet, just submit a Draft PR, so others can contribute to it.

**Reply by Asmageddon** [reply]
Does fzf have a function that just outputs the matches that could be plugged directly into or otherwise integrated with zsh-autocomplete? And/or could its author be contacted about the info?

**Reply by marlonrichert** [reply]
You can contact Fzf Tab’s author here: https://github.com/Aloxaf/fzf-tab/discussions

**Reply by 2KAbhishek** [reply]
Haven't been using this plugin for sometime, was wondering if we had any solutions for this?

@marlonrichert @Asmageddon

**Reply by marlonrichert** [reply]
Yes, this should now be possible. See the [Readme](https://github.com/marlonrichert/zsh-autocomplete/blob/main/README.md#reassign-tab) for more info.

**Comment by marlonrichert** [comment, marked as answer]
This is now fully supported See the [Readme](https://github.com/marlonrichert/zsh-autocomplete/blob/main/README.md#reassign-tab) for more info.
